doc-content/enterprise-only/processes/interacting-with-processes-defining-process-instances-variables-advanced-filters-proc.adoc

```diff
@@ -1,7 +1,7 @@
-[id='interacting-with-processes-process-instances-special-filter-proc']
-= Defining special filters for process instances
+[id='interacting-with-processes-defining-process-instances-variables-advanced-filters-proc']
+= Defining process instance variables using advanced filters
 
-You can use the *Advanced Filters* option in {CENTRAL} to create special filters for process instances. When you create a special filter over the column `processId`, you can view the process instance variables of the process as columns using *Show/hide columns*.
+You can use the *Advanced Filters* option in {CENTRAL} to define process instance variables. When you create a filter over the column `processId`, you can view the process instance variables of the process as columns using *Show/hide columns*.
```

Comment:
Same in here, variables are defined beforehand, this should be more like show or view.

Reply:
Done